### ISSUE TITLE
Web component: Card Section- Carousel - In 300% zoom, we are not able to see the elements and content of the entire page.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4781]
## Detailed description

Issue: 
In 300% zoom, we are not able to see the elements and content of the entire page.

Expected: 
1.Page should be visible with all the elements present on that.
2.Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions.
